src/libs/import.c

```diff
@@ -1961,6 +1961,28 @@ static void _lib_import_from_callback(GtkWidget *widget, dt_lib_module_t* self)
 {
   dt_lib_import_t *d = (dt_lib_import_t *)self->data;
   d->import_case = (widget == GTK_WIDGET(d->import_inplace)) ? DT_IMPORT_INPLACE : DT_IMPORT_COPY;
+
+  if(d->import_case == DT_IMPORT_COPY)
+  {
+    const gboolean setup = dt_conf_get_bool("setup_import_directory");
+    if(setup == FALSE)
+    {
+      gboolean not_again = dt_gui_show_standalone_yes_no_dialog
+            (_("import base directory"),
+             _("Before copying images to the darktable base directory make sure it is defined as you prefer."
+               "\nFurther information can be found in the darktable manual."
+               "\n\nGoto darktable preferences->import."
+               "\nCheck and possibly correct the 'base directory naming pattern'"),
+            _("show this information again"), _("understood & done"));
+      if(not_again)
+        dt_conf_set_bool("setup_import_directory", TRUE);
+      return;
+    }
+  }
+//            (_("Before importing images to the darktable base directory make sure that is set as you prefer.\n"
```

Comment:
Left over? I suspect those 2 lines can be removed.

Reply:
indeed and fixed